app/views/api_docs/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<!doctype html>
+<html>
+  <head>
+    <%= javascript_include_tag "openapi-explorer.min.js" %>
+  </head>
+  <body>
+    <openapi-explorer spec-url="/api/v3/spec.json"> </openapi-explorer>
```

Comment:
There's probably some path helper for `/api/v3/spec.json`. We'll leave that as an exercise for the reader.

Reply:
resolved